File: src/main/java/io/renren/modules/sys/entity/SysUserTokenEntity.java
package io.renren.modules.sys.entity;

import lombok.Data;

import java.io.Serializable;
import java.time.LocalDateTime;


/**
 * 系统用户Token
 */
@Data
public class SysUserTokenEntity implements Serializable {
    private static final long serialVersionUID = 1L;

    //用户ID
    private Long userId;
    //token
    private String token;
    //过期时间
    private LocalDateTime expireTime;
    //更新时间
    private LocalDateTime updateTime;

}

File: src/main/java/io/renren/modules/sys/entity/SysUserTokenEntity.kt
package io.renren.modules.sys.entity

import java.io.Serializable
import java.time.LocalDateTime


/**
 * 系统用户Token
 */
data class SysUserTokenEntity(
        private val serialVersionUID: Long = 1L,
        /**
         * 用户ID
         */
        var userId: Long? = null,
        /**
         * token
         */
        var token: String? = null,
        /**
         * 过期时间
         */
        var expireTime: LocalDateTime? = null,
        /**
         * 更新时间
         */
        var updateTime: LocalDateTime? = null
) : Serializable